unit-tests-jest/numbers.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers) {
+  return numbers.filter((n) => n % 2 === 0);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ * It works for both integer and fractional numbers.
+ * That is, `toDollars(12)` returns `$12.00` and
+ * `toDollars(11.341)` returns `$11.34`.
+ */
+export function toDollars(amount) {
+  if (
+    amount === undefined ||
+    amount === null ||
+    isNaN(amount) ||
+    amount === 0 ||
+    !amount
+  )
+    return 'Invalid value try again';
```

Comment:
It would probably be best to throw an exception instead of returning a string. That way the caller can handle bad input with a try/catch.